sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -35,14 +35,22 @@ def test_operator_to_template(self):
       msg2 = dsl.PipelineParam('msg2', value='value2')
       op = dsl.ContainerOp(name='echo', image='image', command=['sh', '-c'],
                            arguments=['echo %s %s | tee /tmp/message.txt' % (msg1, msg2)],
-                           file_outputs={'merged': '/tmp/message.txt'})
+                           file_outputs={'merged': '/tmp/message.txt'}, gcp_secret='user-gcp-sa')
     golden_output = {
       'container': {
         'image': 'image',
         'args': [
           'echo {{inputs.parameters.msg1}} {{inputs.parameters.msg2}} | tee /tmp/message.txt'
         ],
         'command': ['sh', '-c'],
+        'env': {
```

Comment:
Did you test running this pipeline?

Reply:
Please create the secret in your cluster, compile a pipeline, submit it to the cluster and check for errors.

Reply:
Agree with Alexey for testing it manually before merging